stories/index.js

```diff
@@ -44,7 +44,9 @@ import {
   ListWithOwnItemComponent,
   DefaultLeadModal,
   DefaultCard,
+  DefaultTitle,
 } from './examples'
+import { EmbeddedLinkWithinTitle } from './examples/Title'
```

Comment:
Can't you import this directly from examples like the other ones? Since you are exporting it from the examples index you will have access to it